src/pages/learn.tsx

```diff
@@ -0,0 +1,331 @@
+          title: "Issuer Identity",
+          timeStamp: 634,
+        },
+        {
+          title: "SDK Overview, Code Walk-through",
+          timeStamp: 1300,
+        },
+        {
+          title: "OA-Verify",
+          timeStamp: 1380,
+        },
+        {
+          title: "Document Store Library",
+          timeStamp: 1730,
+        },
+        {
+          title: "Token Registry Library",
+          timeStamp: 1790,
+        },
+      ],
+    },
+  },
+  {
+    title: "Set Up and Change Templates",
+    description:
+      "Building on sessions 2 and 3, this session will walk you through the steps on how to: configure documents; set up schema in a configuration file; and change document templates.",
+    resource: {
+      youtubeEmbedCode: "z3l9OSVGHH8",
+      tag: "Technical",
+      downloads: [
+        {
+          fileName: "TradeTrust Tech Webinar 5.pdf",
+          path: "/static/images/webinar/tradetrust-tech-webinar-5.pdf",
+        },
+      ],
+      videoChapters: [
+        {
+          title: "Introduction to Document Creator",
+          timeStamp: 233,
+        },
+        {
+          title: "Introduction to the Configuration File",
+          timeStamp: 363,
+        },
+        {
+          title: "Setup - transferable document",
+          timeStamp: 641,
+        },
+        {
+          title: "Setup - verifiable document",
+          timeStamp: 1319,
+        },
+        {
+          title: "Issuing document",
+          timeStamp: 1545,
+        },
+        {
+          title: "Verify the newly created document",
+          timeStamp: 1702,
+        },
+      ],
+    },
+  },
+  {
+    title: "Identity Resolution Services",
+    description:
+      "Identity Resolution in TradeTrust involves the ability to map non-readable identification of issuers to a readable form. This segment will cover the methods and the steps on building APIs for identity resolution.",
+    resource: {
+      youtubeEmbedCode: "3S1QjgPXn54",
+      tag: "Technical",
+      downloads: [
+        {
+          fileName: "TradeTrust Tech Webinar 6.pdf",
+          path: "/static/images/webinar/tradetrust-tech-webinar-6.pdf",
+        },
+      ],
+      videoChapters: [
+        {
+          title: "Identity Resolution - Demo",
+          timeStamp: 130,
+        },
+        {
+          title: "Identity Resolution - Responsibilities",
+          timeStamp: 420,
+        },
+        {
+          title: "Identity Resolution - Motivations",
+          timeStamp: 681,
+        },
+        {
+          title: "Panel Discussion",
+          timeStamp: 926,
+        },
+      ],
+    },
+  },
+];
+
+export const LearnPage: FunctionComponent = () => {
+  const postsPerPage = 5;
+  const [currentPage, setCurrentPage] = useState(1);
+  const paginatedPosts = getPaginatedPosts({ posts: webinars, postsPerPage: postsPerPage, currentPage });
+  const totalNoOfPages = getPaginatedPagesTotal({ posts: webinars, postsPerPage: postsPerPage });
+
```

Comment:
extract to another component `Webinars` maybe?

Reply:
Done.